sockeye/lr_scheduler.py

```diff
@@ -128,7 +154,8 @@ def __call__(self, t):
         if self.lr is None:
             assert self.base_lr is not None
             self.lr = self.base_lr
-        return self.lr
+        lr = self._warmup(t) if self.warmup > 0 and t <= self.warmup else self.lr
```

Comment:
not really, since we dont have a t in new_evaluation_result() :)
I can add add a flag 'warmed_up' written to by __call__; its not very elegant though :/